TestProjects/ShaderFoundry/Assets/CommonAssets/Editor/EditorTests/Utilities/ColorExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using UnityEngine;
+
+namespace UnityEditor.ShaderFoundry.UnitTests
+{
+    // These functions were taken from hlsl code
+    internal static class ColorExtensions
+    {
+        public static Color UnpackNormalRGB(this Color packedColor, float scale = 1.0f)
+        {
+            Color normal;
+            normal.r = packedColor.r * 2.0f - 1.0f;
+            normal.g = packedColor.g * 2.0f - 1.0f;
+            normal.b = packedColor.b * 2.0f - 1.0f;
+            normal.r *= scale;
+            normal.b *= scale;
```

Comment:
Is `g` not scaled?

Reply:
No, this is as expected, stolen straight from the shader here: https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.render-pipelines.core/ShaderLibrary/Packing.hlsl#L167